wiki/FAQ/en.md

```diff
@@ -1,731 +1,99 @@
-Each of those points on a slider will add one to the combo.
-Note that the starting circle point is worth 30 points, the slider ticks are worth 10 points each, the reverse arrow points are worth 30 points each, and the end circle point is worth 300 (or geki), 100 (or katu), or 50 points depending on how the slider was completed.
+## Internet Relay Chat
 
-Slider ticks only appears inside its own slider path; if the slider ticks are too close to the start or end circle as shown in the image above, it will be hidden inside the start or end circle.
-Hidden slider ticks still counts into combo and score, so it must be collected along with the visible slider ticks.
-If the slider has one or more reverse arrow points, then the slider ticks will reappear at the same place to be collected in reverse.
+### I am having an error "Bad Authentication Token". What should I do?
 
-If miss the timing to tap the starting circle point, miss the slider ticks, or miss the reverse arrow points, combo will break (it should trigger a combo break sound when the combo is high enough) but it is not considered a miss.
-However, if miss the ending circle point, combo does not break and does not increment by one.
-It is also not considered a miss; instead, a 100 (or katu for end of the combo) will be given for the entire slider, assuming at least the slider starting circle point done properly, and completed all the slider ticks/reverse arrow points when given.
-Notably, [Sudden Death][Sudden Death wikilink] game mod will not trigger its effect of immediate game over when missing the ending circle point (since the combo did not break, the Sudden Death effect was not triggered).
+### Can I use another username?
 
-These situations, caused by a slider, are collectively called **sliderbreak**.
-A max combo play can be known by checking the result screen for the _Perfect_ notice on the performance graph; not having the _Perfect_ notice, especially without any miss, usually means there is one or more sliderbreak occurred during play.
+### What is this voice status that I have? I also see some people having it.
 
-[Answer source with test example in gaming section of stackexchange](https://gaming.stackexchange.com/questions/221663/i-fcd-a-song-why-is-my-combo-lower-than-people-on-the-high-scores)
+## Wiki
 
-### osu!taiko
-#### What does "Use Taiko skin for Taiko mode" in Options under Skin section do?
-The button only works if a folder named `taiko` (capitalisation does not matter) inside the `Skins` folder of the `osu!` file directory existed.
+### Markdown supports HTML tags, can I use them?
 
-When enabled, it will use the `taiko` folder's skin elements when playing in osu!taiko, regardless of current skin's taiko elements.
-If the `taiko` folder was not found, or disabled, it will use the current skin's taiko elements when available; default if all/certain taiko skin element was not found.
+### Why does the file I uploaded not load on the osu!wiki?
```

Comment:
need someone to verifiy but idk if `osu!wiki` is the official term for the wiki or if it is `osu! wiki`